README.md

```diff
@@ -10,4 +10,10 @@ Contains a managed implementation of the WebSocket protocol, along with client a
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
 
 
+## System Requirements
 
+This repo has a few special system requirements/prerequisites.
+
+1. Windows IIS Express tests require IIS Express 10 for WebSockets support
```

Comment:
I'll update it. But isn't that covered by the requirement for IIS Express 10? Or does IIS Express 10 run on Win7? :)